tensorflow/lite/micro/kernels/split_v.cc

```diff
@@ -0,0 +1,257 @@
+/* Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+#include <vector>

```

Comment:
Use of std::vector is usually a red flag on Micro, since it relies on dynamic memory allocation. This does make porting kernels like this tough, but we are open to approaches like capping the maximum number of tensors, like we do with concatenation: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/micro/kernels/concatenation.cc#L31

We do also now have the scratch memory API available if temporary buffers are required, but we try to minimize the memory usage generally.

Reply:
I removed dependencies for std:vector. It's used in ResizeOutputTensors() which we don't need here. The output tensors have already been created the right size directly from the model.

Reply:
I removed all usage of dynamic memory allocation, followed the same flow as in the regular split  operation. No std::vector or any other  dynamic allocation necessary - the output tensors are already allocated in the model.

Reply:
Great! Can you also remove the #include <vector> since it's no longer needed? Thanks.

Reply:
Done :)